src/core/kconfigelektra_p.h

```diff
@@ -0,0 +1,65 @@
+//
+// Created by felix on 28.10.19.
+//
+#include "kconfigbackend_p.h"
```

Comment:
why _p?

Reply:
Header files with the postfix _p are apparently removed from header files and not installed to the include directory. This way the user cannot use internals of the library.